docs/setup/install/install-devtron-helm-3.md

```diff
@@ -1,6 +1,15 @@
-# Install Devtron using Helm3
+# Install Devtron using Helm3 (Deprecated)
 
-To install Helm3, please check [Installing Helm3](https://helm.sh/docs/intro/install/)
+> Info: From Devtron v0.3.27 onwards, using Helm3 to install Devtron will be deprecated.
```

Comment:
Installation using Helm3 is not deprecated. Using Helm2 and kubectl are deprecated. Helm3 is the only way to install devtron now.

We are not removing this page because we don't want users to land on error 404